lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -125,7 +126,7 @@ defmodule LivebookWeb.SessionLive.CellComponent do
           <%= @cell_view.name %>
         </div>
 
-        <.cell_input cell_view={@cell_view} />
+        <.cell_input cell_view={@cell_view} target={@myself} />
```

Comment:
We no longer need this :)

```suggestion
        <.cell_input cell_view={@cell_view} />
```